frontend/src/components/common/authorized-wrapper/authorized-wrapper.tsx

```diff
@@ -9,7 +9,7 @@ const AuthorizedWrapper: FC = ({ children }) => {
       <Header />
       <div className={styles.content}>
         <AuthorizedMenu />
-        {children}
+        <div className={styles.main_content}>{children}</div>
```

Comment:
Classnames in styles file should be in camelCase notion, I think

Reply:
+